onboard/nav/searches.cpp

```diff
@@ -164,6 +164,10 @@ void LawnMower::initializeSearch( Rover* phoebe, const rapidjson::Document& rove
             nextSearchPoint.longitude_deg += totalLongitudeMinutes / 60;
             nextSearchPoint.longitude_min = ( totalLongitudeMinutes - ( ( (int) totalLongitudeMinutes) / 60 ) * 60 );
 
+
```

Comment:
I believe this was from debugging search with Ana, let's remove it since we finished that and likely won't need it in the same way again.